migration-guides/11.0.md

```diff
@@ -59,3 +59,91 @@ The removal the timezone is now an 'opt-in' mechanism triggered by the `x-local-
 
 > [!NOTE]
 > You can revert to the previous behavior thanks to the global property `useLegacyDateExtension`
+<!-- generated deprecated - start -->
```

Comment:
General remarks, do we want to have them all in the same file 
or do we want to do something `depracation-guides/<version>/<package-name>/readme.md` so it's easier to spot if we are impacting by the deprecation or not

Reply:
I would keep them in the same file. Because having a migration file plus a folder where you have version and then package name I think it's not giving a global view of what needs to be done for a migration (even if the deprecations are not mandatory for migration). But this is personal thought, let's see other opinions :)